chapter02/sdt/ex1.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use Cwd qw( abs_path );
+use File::Spec qw( curdir );
+
+opendir(DIR, File::Spec->curdir()) or die $!;
+while (my $file = readdir(DIR)) {
+  print " " x 4 .abs_path($file) . "\n";
+}
+closedir(DIR);
```

Comment:
Normally in a review I'd start talking about file handle references here, but those come later in the book so I'll avoid spoilers.